packages/mds-agency/utils.ts

```diff
@@ -499,7 +499,8 @@ const normalizeTelemetry = (telemetry: TelemetryResult) => {
   return telemetry
 }
 
-export async function readPayload(store: typeof cache | typeof db, device_id: UUID): Promise<VehiclePayload> {
+export async function readPayload(cached: boolean, device_id: UUID): Promise<VehiclePayload> {
+  const store = cached ? cache : db
```

Comment:
I don't like this ... `cache` and `db` don't implement the same set of methods. Using them interchangeably like this is bad practice.